src/main/kotlin/com/dobby/backend/infrastructure/config/SchedulerConfig.kt

```diff
@@ -0,0 +1,28 @@
+package com.dobby.backend.infrastructure.config
+
+import com.dobby.backend.infrastructure.scheduler.ExpiredExperimentPostJob
+import org.quartz.*
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@Configuration
+class SchedulerConfig {
+    @Bean
+    fun expiredExperimentPostJobDetail(): JobDetail {
+        return JobBuilder.newJob(ExpiredExperimentPostJob::class.java)
+            .withIdentity("expired_experiment_post_job")
+            .storeDurably()
+            .build()
+    }
+
+    @Bean
+    fun expiredExperimentPostTrigger(): Trigger {
+        return TriggerBuilder.newTrigger()
+            .forJob(expiredExperimentPostJobDetail())
+            .withIdentity("expired_experiment_post_trigger")
+            .withSchedule(
+                CronScheduleBuilder.dailyAtHourAndMinute(0, 0)
```

Comment:
Quartz 스케줄러는 기본적으로 시스템의 기본 시간대를 따르는데 설정하지 않아도 한국 시간대에 동작할 수 있지만, 안정성과 명시성을 위해 명확히 타임존을 설정하는 것은 어떨까요??
```suggestion
            .withIdentity("expired_experiment_post_trigger")
            .withSchedule(
                CronScheduleBuilder.dailyAtHourAndMinute(0, 0)
                  .inTimeZone(TimeZone.getTimeZone("Asia/Seoul"))
```

Reply:
저도 해당 부분을 저희 '대규모 설계 시스템 기초 스터디'에서 7주차(어제자 발표)로 공부한 TimeZone 이슈에서 명시할 필요성이 있다는 것을 느꼈습니다!
인사이트 있는 피드백 정말 감사합니다! 반영하겠습니다.